File: LeetCode-October-Challenge/ToeplitzMatrix.java
class Solution {
    public boolean isToeplitzMatrix(int[][] matrix) {
        for(int row = 0; row < matrix.length - 1; ++row)
            for(int col = 0; col < matrix[0].length - 1; ++col)
                if(matrix[row][col] != matrix[row + 1][col + 1]) return false;
        return true;
    }
}
public class ToeplitzMatrix {
    public static void main(String[] args) {
        int[][] matrix = {{1,2,3,4},{5,1,2,3},{9,5,1,2}};
        Solution s = new Solution();
        System.out.println(s.isToeplitzMatrix(matrix));
    }
}

File: LeetCode-October-Challenge/ToeplitzMatrix.js
var isToeplitzMatrix = function(matrix) {
    for(let row = 0; row < matrix.length - 1; ++row)
        for(let col = 0; col < matrix[0].length - 1; ++col)
            if(matrix[row][col] !== matrix[row + 1][col + 1]) return false;
    return true;
};
let matrix = [[1,2,3,4],[5,1,2,3],[9,5,1,2]];
console.log(isToeplitzMatrix(matrix));